frontend/src/metabase/admin/upsells/components/UpsellCard.tsx

```diff
@@ -38,9 +54,13 @@ export const _UpsellCard = ({
   });
 
   return (
-    <UpsellCardComponent data-testid="upsell-card">
+    <UpsellCardComponent
+      data-testid="upsell-card"
+      fullWidth={fullWidth}
+      maxWidth={maxWidth}
+    >
       {illustrationSrc && <Image src={illustrationSrc} w="100%" />}
-      <Flex gap="sm" justify="center" p="1rem" pb="0.75rem">
+      <Flex gap="sm" p="1rem" pb="0.75rem">
```

Comment:
The [upsell design](https://www.figma.com/design/kdZZogH2tWBVwnG03N7XXc/Upsell-System?t=WNRRu8GYlBRcGbFm-0) all have the title left-aligned. I think this was mistakenly implemented.